code/style.css

```diff
@@ -1,14 +1,14 @@
 /* Global css variables used for colors */
 :root {
-  --primary: #a259ff;
-  --secondary: #b0a6ff;
+  --primary: black;
+  --secondary: red;
 }
 
 body {
   display: flex;
   margin: 0;
   padding: 0;
-  font-family: 'Montserrat', sans-serif;
+  font-family: "Montserrat", sans-serif;
```

Comment:
I like that you are consistent with the double quotations (") and that you use them cohesively. 

I guess there is a reason behind it? :)